src/pynwb/tests/test_acoustic_waveform_series.py

```diff
@@ -1,58 +1,139 @@
 import numpy as np
-from pynwb.testing import AcquisitionH5IOMixin, TestCase
```

Comment:
What was the reasoning behind removing this?

Reply:
Thanks for pointing that out. I have been trying to move towards using more standard pytest testing but I realize that this mixing is using some methods that I did not re-implement. I reverted.